electrumx/server/session.py

```diff
@@ -213,29 +216,158 @@ def _ssl_context(self):
+                    app.router.add_post('/proxy/blockchain.atomicals.dump', handler.atomicals_dump)
+                    app.router.add_post('/proxy/blockchain.atomicals.at_location', handler.atomicals_at_location)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_location', handler.atomicals_get_location)
+                    app.router.add_post('/proxy/blockchain.atomicals.get', handler.atomicals_get)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_global', handler.atomicals_get_global)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_state', handler.atomical_get_state)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_state_history', handler.atomical_get_state_history)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_events', handler.atomical_get_events)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_tx_history', handler.atomicals_get_tx_history)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_realm_info', handler.atomicals_get_realm_info)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_by_realm', handler.atomicals_get_by_realm)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_by_subrealm', handler.atomicals_get_by_subrealm)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_by_dmitem', handler.atomicals_get_by_dmitem)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_by_ticker', handler.atomicals_get_by_ticker)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_by_container', handler.atomicals_get_by_container)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_by_container_item', handler.atomicals_get_by_container_item)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_by_container_item_validate', handler.atomicals_get_by_container_item_validation)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_container_items', handler.atomicals_get_container_items)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_ft_info', handler.atomicals_get_ft_info)
+                    app.router.add_post('/proxy/blockchain.atomicals.find_tickers', handler.atomicals_search_tickers)
+                    app.router.add_post('/proxy/blockchain.atomicals.find_realms', handler.atomicals_search_realms)
+                    app.router.add_post('/proxy/blockchain.atomicals.find_subrealms', handler.atomicals_search_subrealms)
+                    app.router.add_post('/proxy/blockchain.atomicals.find_containers', handler.atomicals_search_containers)
+                    app.router.add_post('/proxy/blockchain.atomicals.get_holders', handler.atomicals_get_holders)
+                    # common proxy
```

Comment:
This is interesting. Can you elaborate on the reason behind this? Is it needed? 

Wait - I think I get what you have done. You have created an http server on top that then proxies the request underneat to the existing socket session?

And these are "catch all" methods to allow the requests to be passed on?

Please help me follow a bit more ;)

Reply:
Certainly, I have attempted to cover all existing interfaces. If a particular interface is not covered, it will fall back to the common POST and GET interface. This interface attempts to establish a TCP connection and acts as a proxy using the services specified in the TCP configuration.